wmn-data.json

```diff
@@ -3098,6 +3110,18 @@
         "m_code" : 404,
         "known" : ["mohammed01", "mohammed03"],
         "cat" : "gaming",
+        "valid" : true
+       },        
+       {
+        "name" : "likeevideo",
+        "uri_check" : "https://likee.video/@{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "(@",
+        "m_string" : "Unable to find user",

```

Comment:
Looks like this `Unable to find user` is not in the returned HTML but in a JSON file used for errors. Please recheck this and find another `m_string` that is in the HTML.

Reply:
@WebBreacher: `Done!` re change of `m_string` for `likeevideo`